packages/attribute-registry-consumer/src/attributeRegistryConsumerService.ts

```diff
@@ -30,5 +32,11 @@ export async function handleMessage(message: EventEnvelope): Promise<void> {
         { upsert: true }
       );
     })
+    .with({ type: "AttributeDeleted" }, async (msg) => {
```

Comment:
This hander was missing - by centralizing the models and keeping them in sync among consumer and producer, the TS compiler spotted this error 🥳